beamer/agent/relayer.py

```diff
@@ -30,7 +30,7 @@ def run_relayer_for_tx(
     privkey: HexBytes,
     tx_hash: HexBytes,
     prove_tx: bool = False,
-) -> None:
+) -> str | None:
```

Comment:
Suppose we return `None` from this function and there is no exception.
What would happen in `chain.py`?

Reply:
this function is also used by l1 resolution so returning None should be fine.

Reply:
only one prove_tx condition is determining if it is prove or l1 resolution